portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,17 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingcontent() {
+    const responseFromServer = await fetch('/hello');
+    const textdisplayed = await responseFromServer.text();
```

Comment:
Can you use the json parser instead? From the assignment, something like:

// Send a request to /my-data-url.
const responseFromServer = await fetch('/my-data-url');

// Parse the response as JSON.
const myObject = await responseFromServer.json();

Reply:
There are many people that used Gson according to the technical cht room so I feel like my method is fine for now, I know how to do it both ways though.